lib/editor_test.go

```diff
@@ -189,6 +183,7 @@ func TestClipboard(t *testing.T) {
 	}
 }
 
+/* TODO: causing panic
```

Comment:
Maybe `Skip` it rather than commenting it out? https://golang.org/pkg/testing/#T.Skip